File: js/codewars/tests/katas/seven/covfefe.test.js
import { covfefe } from '../../../src/katas/seven/covfefe';

describe('Pruebas en covfefe', () => {
    
    test('Should pass fixed tests', () => {
		expect(covfefe('coverage')).toBe('covfefe');
		expect(covfefe('coverage coverage')).toBe('covfefe covfefe');
		expect(covfefe('nothing')).toBe('nothing covfefe');
		expect(covfefe('double space ')).toBe('double space  covfefe');
		expect(covfefe('covfefe')).toBe('covfefe covfefe');
		expect(covfefe('erag')).toBe('erag covfefe');
	});
});

File: java/codewars/src/test/java/com/codewars/seven/CovfefeTest.java
package com.codewars.seven;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class CovfefeTest {

    @Test
    void testCovfefe() {
        assertEquals("covfefe", Covfefe.covfefe("coverage"));
        assertEquals("covfefe covfefe", Covfefe.covfefe("coverage coverage"));
        assertEquals("nothing covfefe", Covfefe.covfefe("nothing"));
        assertEquals( "double space  covfefe" ,Covfefe.covfefe("double space "));
        assertEquals("covfefe covfefe", Covfefe.covfefe("covfefe"));
        assertEquals("erag covfefe", Covfefe.covfefe("erag"));
    }

}